virttest/utils_test/libvirt.py

```diff
@@ -2092,6 +2092,51 @@ def create_nwfilter_xml(params):
         raise exceptions.TestFail("Fail to create nwfilter XML: %s" % detail)
 
 
+def get_channel_info(session, interface):
+    """
+    Get channel parameters of interface in vm.
+    """
+    cmd = "ethtool -l %s" % interface
+    s, o = session.cmd_status_output(cmd)
+    if s:
+        logging.info("Get channel parameters for vm failed:%s" % o)
```

Comment:
logging.error() is better because this means the function return errors.

Reply:
updated, Thank you!